faculty-affiliations.csv

```diff
@@ -8798,9 +8802,11 @@ Gena Hahn , University of Montreal
 Gilles Brassard , University of Montreal
 Guy Lapalme , University of Montreal
 Houari A. Sahraoui , University of Montreal
+Ioannis (Yannis) Mitliagkas , University of Montreal
```

Comment:
This is not the correct DBLP entry. Please ensure that all entries correspond to the faculty name(s) given at [DBLP](http://dblp.org).

Reply:
Sure ..